src/audio/copier/copier.c

```diff
@@ -610,25 +610,23 @@ static int copier_prepare(struct comp_dev *dev)
 	if (ret == COMP_STATUS_STATE_ALREADY_SET)
 		return PPL_STATUS_PATH_STOP;
 
-	if (cd->endpoint_num) {
+	for (i = 0; i < cd->endpoint_num; i++) {
```

Comment:
there's a similar check on line 1010. Can you please fix that too?

Reply:
and please lets be respectful in commit messages. maybe say remove redundant assignment instead?

Reply:
> there's a similar check on line 1010. Can you please fix that too?

That should be done as a separate commit (and might be done as a separate PR). The purpose of this commit is just to remove "cd->converter[i] = cd->converter[i];" redundant assignment.

Reply:
> and please lets be respectful in commit messages. maybe say remove redundant assignment instead?

You are right, "redundant" is more appropriate. Re-edited commit message.